### DISCUSSION TITLE
How to use GitHub CLI inside a Codespaces?

### DISCUSSION THREAD

**Discussion by haifeng-jin** [community/community#41238] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I am trying to use GitHub CLI inside my devcontainer in Codespaces.
I have install it myself.

Is it possible to always have the latest GitHub CLI installed when a Codespace is started?

**Comment by thomas-sickert** [comment, marked as answer]
Hey @haifeng-jin, you can set up your devcontainer to have the GitHub CLI installed using [features](https://code.visualstudio.com/blogs/2022/09/15/dev-container-features#_adding-features-to-your-dev-container). The GitHub CLI feature can be found [here](https://github.com/devcontainers/features/tree/main/src/github-cli).

**Reply by haifeng-jin** [reply]
Thank you! I just tried. It works great!
